apps/main/src/entities/gauge/model.ts

```diff
@@ -0,0 +1,99 @@
+import { queryOptions } from '@tanstack/react-query'
+import * as api from './api'
+import { gaugeKeys as keys } from './query-keys'
+import { REFRESH_INTERVAL } from '@/constants'
+import { queryClient } from '@/shared/api/query-client'
+import { zeroAddress } from 'viem'
+import type { AddRewardTokenParams } from './types'
+
+export const getGaugeStatusQuery = (chain: ChainId, poolId: string) =>
+  queryOptions({
+    queryKey: keys.status(chain, poolId),
```

Comment:
queryKey defines a unique data key that we will always get an idempotent result. If we add a wallet address, it will be unnecessary data, because for different wallets and in its absence we will always get the same result on the sent call